### PR TITLE
📝(dimai) update docs on dimail

### PR DESCRIPTION
**Pull request by mjeammet**
## Purpose

update docs on dimail to describe interoperability between la régie and dimail

## Proposal

Description...

- [x] write doc on interop between people and dimail
- [x] update README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -71,6 +65,23 @@ $ make superuser
...
+
+```bash
+$ make demo
+```
+
+### Setting dimail database 
+
+To ease local development when working on interoperability between people and dimail, we embark dimail-api in a container running in "fake" mode. 
+
+To fill dimail local database with users/domains/permissions needed for basic development:
```
Comment (Morendil) [inline README.md:80]:
```suggestion
To populate dimail local database with users/domains/permissions needed for basic development:
```

Slightly pedantic suggestion…

File: docs/interoperability/dimail.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+### Users
+
+The ones issuing requests. Dimail users will reflect domains owners and administrators on People, for logging purposes and to allow direct use of dimail, if desired. User reconciliation is made on user uuid provided by ProConnect.
+
+### Accesses
+
+As for People, an access - a permissions (or "allows" in dimail) - grants an user permission to create objects on a domain.
+
+Permissions requests are sent is automatically created upon :
```
Comment (Morendil) [inline docs/interoperability/dimail.md:26]: I'm failing to parse "are sent is" in this sentence.

**Review by Morendil — APPROVED** [review]
Two minor nitpicks, but since it's documentation, looks very good as-is !

File: docs/interoperability/dimail.md (hunk 2 above)
Comment (mjeammet) [inline docs/interoperability/dimail.md:26]:
```suggestion
Permissions requests are sent automatically upon : 
```

**Pull request merged**